docs/containers.conf.5.md

```diff
@@ -0,0 +1,339 @@
+  Specified as 'device-on-host:device-on-container:permissions',
+  for example: "--additional-devices=/dev/sdc:/dev/xvdc:rwm".
+
+**additional_volumes**=[]
+  List of additional volumes.
+  Specified as "directory-on-host:directory-in-container:options",
+  for example:  "--aditional-volumes=/db:/var/lib/db:ro".
+
+**apparmor_profile**=""
+  Used to change the name of the default AppArmor profile of container engines.
+The default profile name is "container-default".
+
+**cgroupns**="private"
+  Default way to to create a cgroup namespace for the container
+  Options are:
+    `private` Create private Cgroup Namespace for the container.
+    `host`    Share host Cgroup Namespace with the container.
+
+**cgroup_manager**="systemd"
+  The cgroup management implementation used for the runtime. Supports `cgroupfs`
+and `systemd`.
+
+**default_capabilities**=[]
+  List of default capabilities for containers.
+
+  The default list is:
+```
+  default_capabilities = [
+	"AUDIT_WRITE",
+	      "CHOWN",
+	      "DAC_OVERRIDE",
+	      "FOWNER",
+	      "FSETID",
+	      "KILL",
+	      "MKNOD",
+	      "NET_BIND_SERVICE",
+	      "NET_RAW",
+	      "SETGID",
+	      "SETPCAP",
+	      "SETUID",
+	      "SYS_CHROOT",
+  ]
+```
+
+**default_sysctls**=[]
+  List of default sysctls.
+
+**default_ulimits**=[]
+  A list of ulimits to be set in containers by default,
+  specified as "ulimit name=soft-limit:hard-limit",
+  for example:"nofile=1024:2048".
+
+**dns_options**=[]
+  List of default DNS options to be added to /etc/resolv.conf inside of the
+container.
+
+**dns_searches**=[]
+  List of default DNS search domains to be added to /etc/resolv.conf inside of
+the container.
+
+**dns_servers**=[]
+  A list of dns servers to override the DNS configuration passed to the
+container. The special value “none” can be specified to disable creation of
+/etc/resolv.conf in the container.
+
+**env**=["PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"]
+  Environment variable list for the container process, used for passing
+environment variables to the container.
+
+**env_host**=false
+  Pass all host environment variables into the container.
+
+**hooks_dir**=["/etc/containers/oci/hooks.d", ...]
+  Path to the OCI hooks directories for automatically executed hooks.
+
+**http_proxy**=[]"`
+  The proxy environment variable list to apply to container process
+
+**init**=false
+  Run an init inside the container that forwards signals and reaps processes.
+
+**init_path**="/usr/libexec/podman/catatonit"
+  Path to the container-init binary, which forwards signals and reaps processes
+within containers.  Note that the container-init binary will only be used when
+the `--init` for podman-create and podman-run is set.
+
+
+**ipcns**="private"
+  Default way to to create a IPC namespace for the container
+  Options are:
+    `private` Create private IPC Namespace for the container.
+    `host`    Share host IPC Namespace with the container.
```

Comment:
This (as well below) might need list itemization. I _think_ that´s done with a minus